src/code/templates/fv-egg-sort-game.js

```diff
@@ -230,9 +230,10 @@ export default class FVEggSortGame extends Component {
     animation: null,
     animatedComponents: [],
     eggs: []
-  }
+  };
 
-  static backgroundClasses = 'fv-layout fv-layout-d'
+  static backgroundClasses = 'fv-layout fv-layout-d';
+  static maxScore = 8;
```

Comment:
I think it would be better to have this be state that is initialized by the template rather than static data. Given the fact that the egg sort game will also need to count `incorrect` responses, it might make sense to have a `score` object in the state that has properties for `correct`, `incorrect`, and 'maxScore`. Then if other templates require additional scoring options, there'd be a logical place to put them.